### DISCUSSION TITLE
Multiple Plan_IDs allowed in single Allowed Amount MRF?

### DISCUSSION THREAD

**Discussion by nhartmanNWA** [CMSgov/price-transparency-guide#219] [Q&A] [answered]
If a group health plan (ACME) has multiple plans (Plan A, Plan B, Plan C), would the group health plan create a single allowed amount file that contains multiple Plan_IDs and the associated data, or would there need to be an allowed amount file per Plan*, resulting in multiple allowed amount files for the group health plan?  Thank you.

**Comment by ericsilvertx** [comment, marked as answer]
I think there was a recent change in the schema which allows you to put in the header more than one plan reference.  The change to the schema was published last week

**Comment by Devyandu** [comment]
If you look at the [file naming convention ](https://github.com/CMSgov/price-transparency-guide#file-naming-convention) you will see that the plan has to produce a separate allowed amount file for each plan id.

However, to avoid the situation of nearly identical files for multiple plans, there has been a recent change where if two of your plans **have identical pricing contracts** then you can produce a single file for both the plans and reference both the plans ids in the header section.

**Reply by nhartmanNWA** [reply]
Thank you.  If a file has multiple plans referenced, how does that impact the file naming convention?
